deploy/dracon/values/dev.yaml

```diff
@@ -32,6 +32,8 @@ image:
 
 postgresql:
   enabled: true
+
+database:
```

Comment:
This missing was causing a nil value error thrown by Helm. Fixed this with the help of @ptzianos